### PR TITLE
Fix wording around specification samples

### PR DESCRIPTION
**Pull request by qidane**
I also validated the sample code looking for errors, none found.

### CODE REVIEW

File: formats/specifications/compliant-hcs.txt [hunk 1]
```diff
@@ -83,6 +83,8 @@ Alternative valid forms:
 -  would have a ``ome:TiffData`` block instead of the ``bf:BinData`` block 
    (this would be used in the header of an OME-TIFF file)
 -  would have a ``ome:MetadataOnly`` block instead of the ``bf:BinData`` 
-   block (this would **ONLY** be acceptable if the image data of the file
-   **CANNOT** be represented in an OME file for technical reasons)
+   block (this would be used as a companion to one or more ``ome:BinaryOnly``
+   OME-TIFF file)
```
Comment (hflynn) [inline formats/specifications/compliant-hcs.txt]: files).

File: formats/specifications/index.txt [hunk 2]
```diff
@@ -154,8 +154,11 @@ Alternative valid forms:
 -  would have a ``OME:TiffData`` block instead of the ``BIN:BinData`` block
    (this would be used in the header of an OME-TIFF file)
 -  would have a ``OME:MetadataOnly`` block instead of the ``BIN:BinData``
-   block (this would **ONLY** be acceptable if the image data of the file
-   **CANNOT** be represented in an OME file for technical reasons)
+   block (this would be used as a companion to one or more ``OME:BinaryOnly``
+   OME-TIFF file)
```
Comment (hflynn) [inline formats/specifications/index.txt]: files).

File: formats/specifications/minimum.txt [hunk 3]
```diff
@@ -43,5 +43,8 @@ Alternative valid forms:
 -  would have a ``<ome:TiffData/>`` block instead of the ``bf:BinData``
    block (this would be used in the header of an OME-TIFF file)
 -  would have a ``<ome:MetadataOnly/>`` block instead of the ``bf:BinData``
-   block (this would be used in the header of a metadata-only OME file)
+   block (this would be used as a companion to one or more ``ome:BinaryOnly``
+   OME-TIFF file)
```
Comment (hflynn) [inline formats/specifications/minimum.txt]: files)

**Comment by hflynn** [comment]
Aside from the minor nitpick of files needing to be plural in this case, all renders fine on staging. Does this require technical review by someone else?

**Comment by qidane** [comment]
Should be fine without technical review.

**Comment by hflynn** [comment]
Ok, thanks. I'll merge once the builds are green.

**Comment by hflynn** [comment]
--no-rebase

**Pull request merged**